src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Files.kt

```diff
@@ -68,11 +68,13 @@ class Files : FileSystem
         return if (file.extension == "url")
         {
             val url = file.readText().trim('\n').trim().split("URL=").last()
-            DocumentDetails(url, file.absolutePath, file.absolutePath.removePrefix(documentsRoot), true, true)
+            val name = file.name.substringBefore(".") // don't include extension in name
```

Comment:
Is it worth taking the substring up to the last occurrence of '.' rather than the first as here?

Reply:
That would result in a name like "Burden report HepB.web"

Reply:
Yeah, is that how they generally look then? With a '.web.url' suffix? I just wondered if there would ever be a '.' in the meaningful part of file name before the final .url e.g. ''Diagnostic report from montagu.vaccineimpact.org" or something. If not then fair enough, go ahead and merge.

Reply:
hmm. could explicitly strip the ".web.url" bit instead. I'll do that